### PR TITLE
aflplusplus_ddisasm

### PR DESCRIPTION
**Pull request by vnatarajan-gt**
Compare ddisasm with other fuzzers.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
> Compare ddisasm with other fuzzers.

Is there a paper or writeup somewhere discussing ddisasm?

**Comment by jonathanmetzman** [comment]
Please sign the CLA btw

**Comment by vnatarajan-gt** [comment]
Apologies, I'd like to delete this pull request and create another one from the company's github.

File: fuzzers/aflplusplus_ddisasm/builder.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
```
[the remark this answers is not in the document]
Comment (vnatarajan-gt) [reply fuzzers/aflplusplus_ddisasm/builder.Dockerfile:1]: Let me make those changes, I'd like to close this pull request and keep the other one open if it's all right.

**Pull request closed without merging**